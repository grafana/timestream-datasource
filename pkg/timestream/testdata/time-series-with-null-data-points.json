{
    "ColumnInfo": [
        {
            "Name": "interpolatedValue",
            "Type": {
                "ArrayColumnInfo": null,
                "RowColumnInfo": null,
                "ScalarType": null,
                "TimeSeriesMeasureValueColumnInfo": {
                    "Name": null,
                    "Type": {
                        "ArrayColumnInfo": null,
                        "RowColumnInfo": null,
                        "ScalarType": "DOUBLE",
                        "TimeSeriesMeasureValueColumnInfo": null
                    }
                }
            }
        }
    ],
    "NextToken": null,
    "QueryId": "#QueryId#",
    "QueryStatus": {
        "CumulativeBytesMetered": 10000000,
        "CumulativeBytesScanned": 0,
        "ProgressPercentage": 100
    },
    "Rows": [
        {
            "Data": [
                {
                    "ArrayValue": null,
                    "NullValue": true,
                    "RowValue": null,
                    "ScalarValue": null,
                    "TimeSeriesValue": null
                }
            ]
        }
    ]
}