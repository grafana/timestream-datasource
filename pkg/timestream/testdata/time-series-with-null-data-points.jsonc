//  ðŸŒŸ This was machine generated.  Do not edit. ðŸŒŸ
//  
//  Frame[0] {
//      "typeVersion": [
//          0,
//          0
//      ]
//  }
//  Name: 
//  Dimensions: 2 Fields by 0 Rows
//  +-------------------+-------------------------+
//  | Name: time        | Name: interpolatedValue |
//  | Labels:           | Labels:                 |
//  | Type: []time.Time | Type: []*float64        |
//  +-------------------+-------------------------+
//  +-------------------+-------------------------+
//  
//  
//  ðŸŒŸ This was machine generated.  Do not edit. ðŸŒŸ
{
  "status": 200,
  "frames": [
    {
      "schema": {
        "meta": {
          "typeVersion": [
            0,
            0
          ]
        },
        "fields": [
          {
            "name": "time",
            "type": "time",
            "typeInfo": {
              "frame": "time.Time"
            }
          },
          {
            "name": "interpolatedValue",
            "type": "number",
            "typeInfo": {
              "frame": "float64",
              "nullable": true
            },
            "labels": {}
          }
        ]
      },
      "data": {
        "values": [
          [],
          []
        ]
      }
    }
  ]
}