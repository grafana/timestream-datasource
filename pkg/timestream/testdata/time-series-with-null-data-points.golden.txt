ðŸŒŸ This was machine generated.  Do not edit. ðŸŒŸ

Frame[0] {}
Name: 
Dimensions: 2 Fields by 0 Rows
+-------------------+-------------------------+
| Name: time        | Name: interpolatedValue |
| Labels:           | Labels:                 |
| Type: []time.Time | Type: []*float64        |
+-------------------+-------------------------+
+-------------------+-------------------------+


====== TEST DATA RESPONSE (arrow base64) ======
FRAME=QVJST1cxAAD/////2AEAABAAAAAAAAoADgAMAAsABAAKAAAAFAAAAAAAAAEEAAoADAAAAAgABAAKAAAACAAAAHQAAAADAAAATAAAACgAAAAEAAAAuP7//wgAAAAMAAAAAAAAAAAAAAAFAAAAcmVmSWQAAADY/v//CAAAAAwAAAAAAAAAAAAAAAQAAABuYW1lAAAAAPj+//8IAAAADAAAAAIAAAB7fQAABAAAAG1ldGEAAAAAAgAAAMQAAAAYAAAAAAASABgAFAATABIADAAAAAgABAASAAAAFAAAAGwAAABsAAAAAAADAWwAAAACAAAAOAAAAAQAAABc////CAAAABwAAAARAAAAaW50ZXJwb2xhdGVkVmFsdWUAAAAEAAAAbmFtZQAAAACM////CAAAAAwAAAACAAAAe30AAAYAAABsYWJlbHMAAAAAAAB+////AAACABEAAABpbnRlcnBvbGF0ZWRWYWx1ZQASABgAFAAAABMADAAAAAgABAASAAAAFAAAAEQAAABMAAAAAAAACkwAAAABAAAADAAAAAgADAAIAAQACAAAAAgAAAAQAAAABAAAAHRpbWUAAAAABAAAAG5hbWUAAAAAAAAAAAAABgAIAAYABgAAAAAAAwAEAAAAdGltZQAAAAAQAAAADAAUABIADAAIAAQADAAAABAAAAAUAAAAJAAAAAAABAAAAAAAAAAAAAAAAAAAAAAAAAAKAAwAAAAIAAQACgAAAAgAAAB0AAAAAwAAAEwAAAAoAAAABAAAALj+//8IAAAADAAAAAAAAAAAAAAABQAAAHJlZklkAAAA2P7//wgAAAAMAAAAAAAAAAAAAAAEAAAAbmFtZQAAAAD4/v//CAAAAAwAAAACAAAAe30AAAQAAABtZXRhAAAAAAIAAADEAAAAGAAAAAAAEgAYABQAEwASAAwAAAAIAAQAEgAAABQAAABsAAAAbAAAAAAAAwFsAAAAAgAAADgAAAAEAAAAXP///wgAAAAcAAAAEQAAAGludGVycG9sYXRlZFZhbHVlAAAABAAAAG5hbWUAAAAAjP///wgAAAAMAAAAAgAAAHt9AAAGAAAAbGFiZWxzAAAAAAAAfv///wAAAgARAAAAaW50ZXJwb2xhdGVkVmFsdWUAEgAYABQAAAATAAwAAAAIAAQAEgAAABQAAABEAAAATAAAAAAAAApMAAAAAQAAAAwAAAAIAAwACAAEAAgAAAAIAAAAEAAAAAQAAAB0aW1lAAAAAAQAAABuYW1lAAAAAAAAAAAAAAYACAAGAAYAAAAAAAMABAAAAHRpbWUAAAAA8AEAAEFSUk9XMQ==
